assets/js/base/components/payment-methods/payment-methods.js

```diff
@@ -143,7 +142,8 @@ const PaymentMethods = () => {
 						typeof label === 'string'
 							? label
 							: cloneElement( label, {
-									components: { PaymentMethodIcons },
+									activePaymentMethod,
+									...currentPaymentMethodInterface.current,
```

Comment:
I'm not sure we want to expose the entire interface to the label component. We should just expose components:

```suggestion
									components: { ...currentPaymentMethodInterface.current.components },
```